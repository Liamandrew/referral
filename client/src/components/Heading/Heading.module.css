.heading {
    font-size: 38px;
    font-weight: 900;
    color: #0d0d0d;
}
