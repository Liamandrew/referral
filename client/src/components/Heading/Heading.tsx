import React from "react";
import style from "./Heading.module.css";

const Heading: React.FC = ({ children }) => (
    <h1 className={style.heading}>{children}</h1>
);

export { Heading };
