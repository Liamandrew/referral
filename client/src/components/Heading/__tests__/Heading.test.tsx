import React from "react";
import { render } from "@testing-library/react";
import { Heading } from "../Heading";

describe("<Heading>", () => {
    test("should match snapshot", () => {
        const { container } = render(<Heading>text</Heading>);

        expect(container).toMatchSnapshot();
    });
});
