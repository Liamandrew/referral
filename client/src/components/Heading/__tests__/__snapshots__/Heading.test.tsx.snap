// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading> should match snapshot 1`] = `
<div>
  <h1
    class="heading"
  >
    text
  </h1>
</div>
`;
